app/api/odm/specs/logHelper.spec.ts

```diff
@@ -19,6 +19,7 @@ describe('NoLogger', () => {
   it('should return empty promises for logging methods', async () => {
     const logHelper = createUpdateLogHelper('activitylog');
     await expect(logHelper.getAffectedIds(undefined)).resolves.toBeEmpty;
+    // @ts-ignore
     await expect(logHelper.upsertLogMany({ _id: null })).resolves.toBeEmpty;
```

Comment:
This is testing a value that is not valid as _id.  This type of errors should, in theory, be caught by TS from now on, so this test scenario should ideally not arise.  Unsure if this merits a deletion of both expects (this and the next one, which is more or less the same situation), but I'm keeping them as I am unsure of the scenario this was written in.